pymc/model/fgraph.py

```diff
@@ -391,6 +393,11 @@ def clone_model(model: Model) -> Model:
             z = pm.Deterministic("z", clone_x + 1)
 
     """
+    check_for_gp_vars = [
```

Comment:
should be in the even lower fgraph_from_model

Reply:
Oh! my bad, I will fix that now. Thank you @ricardoV94!